### DISCUSSION TITLE
Phasefilt error in dealing Turkey earthquake

### DISCUSSION THREAD

**Discussion by BelledDeng** [gmtsar/gmtsar#690] [Q&A]
Dear Xiaohua-Eric-Xu,
Recently I am using p2p_S1_TOPS_Frame.csh to process the Sentinel-1 data to get the 2023.02.06 Mw7.8 Turkey earthquake coseismic LOS deformation. However,whatever I used ascending or descending track SLC data, the program always reported the same errors(File ../F1/intf/2023028_2023040/phasefilt.grd not found).I guess it maybe the GMTSAR has some difficulty in phasefit processing when dealing Turkey where exists large body of water.Could you tell me what shall I do?

ERROR report:
gmtsar-user@gmtsarVB:/home/sbas/Turkipe_7.8_2023.02.06/descending/sourth$ p2p_S1_TOPS_Frame.csh S1A_IW_SLC__1SDV_20230129T033452_20230129T033519_046993_05A2FE_BE0B.SAFE S1A_OPER_AUX_POEORB_OPOD_20230218T080751_V20230128T225942_20230130T005942.EOF S1A_IW_SLC__1SDV_20230210T033451_20230210T033518_047168_05A8CD_E5B0.SAFE S1A_OPER_AUX_POEORB_OPOD_20230302T080809_V20230209T225942_20230211T005942.EOF config.s1a.txt vv 1
vv
1
Linking files for Subswath 1 ...
Linking files for Subswath 2 ...
Linking files for Subswath 3 ...
[1] 26690
[2] 26691
[3] 26692
[3]  - Done                   p2p_processing.csh S1_TOPS  ...
[2]  - Done                   p2p_processing.csh S1_TOPS  ...
[1]  + Done                   p2p_processing.csh S1_TOPS  ...

Merging START
Number of Files to be merged is 3 
Reading in the grids...
merge_swath [ERROR]: File ../F1/intf/2023028_2023040/phasefilt.grd not found
[Session merge_swath (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grids ../F1/intf/2023028_2023040/phasefilt.grd 
Number of Files to be merged is 3 
Reading in the grids...
merge_swath [ERROR]: File ../F1/intf/2023028_2023040/corr.grd not found
[Session merge_swath (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grids ../F1/intf/2023028_2023040/corr.grd 
Number of Files to be merged is 3 
Reading in the grids...
merge_swath [ERROR]: File ../F1/intf/2023028_2023040/mask.grd not found
[Session merge_swath (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grids ../F1/intf/2023028_2023040/mask.grd 
Merging END

Recomputing the projection LUT...
grep: S1_20230129_033452_F1.PRM: No such file or directory

 error -  could not open file S1_20230129_033452_F1.PRM 

couldn't open master.PRM 
 
 Usage: SAT_llt2rat master.PRM prec [-bo[s|d]] < inputfile > outputfile  

             master.PRM   -  parameter file for master image and points to LED orbit file 
             precise      -  (0) standard back geocoding, (1) - polynomial refinenent (slower) 
             inputfile    -  lon, lat, elevation [ASCII] 
             outputfile   -  range, azimuth, elevation(ref to radius in PRM), lon, lat [ASCII default] 
             -bos or -bod -  binary single or double precision output (only output results within 
                             data coverage, PRM num_lines, num_rng_bins ) 
 
 example: SAT_llt2rat master.PRM 0 < topo.llt > topo.ratll    

 error -  could not open file S1_20230129_033452_F1.PRM 

grdinfo [ERROR]: Error for input file: No such file (phasefilt.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files

SKIP UNWRAP PHASE
rm: No match.
../../F1/iono_phase/intf_h/: No such file or directory.
cp: No match.
../../F1/iono_phase/intf_l/: No such file or directory.
cp: No match.
../../F1/iono_phase/intf_o/: No such file or directory.
cp: No match.
ls: No match.
ls: No match.
grep: ../intf_h/params1: No such file or directory
grep: ../intf_h/params1: No such file or directory
grep: ../intf_h/params1: No such file or directory
Applying split spectrum result to estimate ionospheric phase ( )...
cp: cannot stat '../../merge/phasefilt.grd': No such file or directory
grep: ../intf_h/: Is a directory
grep: ../intf_h/: Is a directory
grep: ../intf_h/: Is a directory
grdinfo [ERROR]: Error for input file: No such file (../intf_h/phasefilt.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
grdinfo [ERROR]: Error for input file: No such file (../intf_h/phasefilt.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
Filtering size is set to -nan along range and -nan along azimuth ...
cp: cannot stat '../intf_h/unwrap.grd': No such file or directory
cp: cannot stat '../intf_l/unwrap.grd': No such file or directory
cp: cannot stat '../intf_o/unwrap.grd': No such file or directory
grdmath [ERROR]: Syntax error: up_h.grd is not a number, operator or file name
grdinfo [ERROR]: Error for input file: No such file (tmp.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
Correcting high passed phase by  * 2PI ...
grdmath [ERROR]: Syntax error: up_h.grd is not a number, operator or file name
mv: cannot stat 'tmp.grd': No such file or directory
grdmath [ERROR]: Syntax error: up_l.grd is not a number, operator or file name
grdinfo [ERROR]: Error for input file: No such file (tmp.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
Correcting high passed phase by  * 2PI ...
grdmath [ERROR]: Syntax error: up_l.grd is not a number, operator or file name
mv: cannot stat 'tmp.grd': No such file or directory
grdmath [ERROR]: Syntax error: up_h.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: tmp.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: up_h.grd is not a number, operator or file name
grdfilter [ERROR]: Error for input file: No such file (tmp.grd)
grdfilter [ERROR]: Syntax error: Must specify input file
grdmath [ERROR]: Syntax error: up_l.grd is not a number, operator or file name
grdfilter [ERROR]: Error for input file: No such file (tmp.grd)
grdfilter [ERROR]: Syntax error: Must specify input file
grdmath [ERROR]: Syntax error: ../intf_h/corr.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: ../intf_h/corr.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: mask1.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: up_l.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: tmp_ph0.grd is not a number, operator or file name
cp: cannot stat 'tmp_ph.grd': No such file or directory
grdinfo [ERROR]: Error for input file: No such file (tmp_ph1.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
grdmath [ERROR]: Syntax error: tmp_ph0.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: tmp_ph0.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: tmp1.grd is not a number, operator or file name
mv: cannot stat 'tmp.grd': No such file or directory
grdmath [ERROR]: Syntax error: tmp1.grd is not a number, operator or file name
mv: cannot stat 'tmp.grd': No such file or directory
grdmath [ERROR]: Syntax error: tmp_ph0.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: mask1.grd is not a number, operator or file name
Reading in original grid...
nearest_grid [ERROR]: File tmp_ph.grd not found
[Session nearest_grid (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grdfile tmp_ph.grd 
grdfilter [ERROR]: Error for input file: No such file (tmp_ph_interp.grd)
grdfilter [ERROR]: Unable to decode -na as a floating point number
surface [ERROR]: Could not parse tmp_ph0.grd into Geographical, Cartesian, or Temporal coordinates!surface [ERROR]: Syntax error -R option. Correct syntax:
	-R<xmin>/<xmax>/<ymin>/<ymax>[/<zmin>/<zmax>]
	  Append r if giving lower left and upper right coordinates
	-Rg or -Rd for global domain
	-R<grdfile> to take the domain from a grid file
surface [ERROR]: Offending option -Rtmp_ph0.grd
grd2xyz [ERROR]: File tmp_filt.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
mv: cannot stat 'tmp.grd': No such file or directory
cp: cannot stat 'tmp_filt.grd': No such file or directory
grdmath [ERROR]: Syntax error: tmp_filt.grd is not a number, operator or file name
Reading in original grid...
nearest_grid [ERROR]: File tmp.grd not found
[Session nearest_grid (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grdfile tmp.grd 
grdmath [ERROR]: Syntax error: tmp2.grd is not a number, operator or file name
grdfilter [ERROR]: Error for input file: No such file (tmp_ph_interp.grd)
grdfilter [ERROR]: Unable to decode -na as a floating point number
surface [ERROR]: Could not parse tmp_ph0.grd into Geographical, Cartesian, or Temporal coordinates!surface [ERROR]: Syntax error -R option. Correct syntax:
	-R<xmin>/<xmax>/<ymin>/<ymax>[/<zmin>/<zmax>]
	  Append r if giving lower left and upper right coordinates
	-Rg or -Rd for global domain
	-R<grdfile> to take the domain from a grid file
surface [ERROR]: Offending option -Rtmp_ph0.grd
grd2xyz [ERROR]: File tmp_filt.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
mv: cannot stat 'tmp.grd': No such file or directory
cp: cannot stat 'tmp_filt.grd': No such file or directory
grdmath [ERROR]: Syntax error: tmp_filt.grd is not a number, operator or file name
Reading in original grid...
nearest_grid [ERROR]: File tmp.grd not found
[Session nearest_grid (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grdfile tmp.grd 
grdmath [ERROR]: Syntax error: tmp2.grd is not a number, operator or file name
grdfilter [ERROR]: Error for input file: No such file (tmp_ph_interp.grd)
grdfilter [ERROR]: Unable to decode -na as a floating point number
surface [ERROR]: Could not parse tmp_ph0.grd into Geographical, Cartesian, or Temporal coordinates!grd2xyz [ERROR]: File tmp_filt.grd not found
surface [ERROR]: Syntax error -R option. Correct syntax:
	-R<xmin>/<xmax>/<ymin>/<ymax>[/<zmin>/<zmax>]
	  Append r if giving lower left and upper right coordinates
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
	-Rg or -Rd for global domain
	-R<grdfile> to take the domain from a grid file
surface [ERROR]: Offending option -Rtmp_ph0.grd
mv: cannot stat 'tmp.grd': No such file or directory
cp: cannot stat 'tmp_filt.grd': No such file or directory
grdmath [ERROR]: Syntax error: tmp_filt.grd is not a number, operator or file name
Reading in original grid...
nearest_grid [ERROR]: File tmp.grd not found
[Session nearest_grid (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grdfile tmp.grd 
grdmath [ERROR]: Syntax error: tmp2.grd is not a number, operator or file name
grdinfo [ERROR]: Error for input file: No such file (ph0.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
grdinfo [ERROR]: Error for input file: No such file (tmp_ph0.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
grdfilter [ERROR]: Error for input file: No such file (tmp_ph_interp.grd)
grdfilter [ERROR]: Unable to decode -na as a floating point number
surface [ERROR]: Syntax error: Must specify -R option
surface [ERROR]: Syntax error -I option: Must specify positive increment(s)
grd2xyz [ERROR]: File tmp_filt.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
mv: cannot stat 'tmp.grd': No such file or directory
grdmath [ERROR]: Syntax error: tmp_filt.grd is not a number, operator or file name
cp: cannot stat 'tmp_ph.grd': No such file or directory
grdsample [ERROR]: Could not parse ph0.grd into Geographical, Cartesian, or Temporal coordinates!grdsample [ERROR]: Syntax error -R option. Correct syntax:
	-R<xmin>/<xmax>/<ymin>/<ymax>[/<zmin>/<zmax>]
	  Append r if giving lower left and upper right coordinates
	-Rg or -Rd for global domain
	-R<grdfile> to take the domain from a grid file
grdsample [ERROR]: Offending option -Rph0.grd
grdmath [ERROR]: Syntax error: ph0.grd is not a number, operator or file name
grdinfo [ERROR]: Error for input file: No such file (ph_corrected.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
Correcting iono phase by  PI ...
grdmath [ERROR]: Syntax error: tmp_filt.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: tmp_ph.grd is not a number, operator or file name
grdsample [ERROR]: Could not parse ph0.grd into Geographical, Cartesian, or Temporal coordinates!grdsample [ERROR]: Syntax error -R option. Correct syntax:
	-R<xmin>/<xmax>/<ymin>/<ymax>[/<zmin>/<zmax>]
	  Append r if giving lower left and upper right coordinates
	-Rg or -Rd for global domain
	-R<grdfile> to take the domain from a grid file
grdsample [ERROR]: Offending option -Rph0.grd
grdmath [ERROR]: Syntax error: ph0.grd is not a number, operator or file name
mv: cannot stat 'tmp_ph.grd': No such file or directory

Merging START
Number of Files to be merged is 3 
Reading in the grids...
merge_swath [ERROR]: File ../F1/intf/2023028_2023040/phasefilt.grd not found
[Session merge_swath (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grids ../F1/intf/2023028_2023040/phasefilt.grd 
Number of Files to be merged is 3 
Reading in the grids...
merge_swath [ERROR]: File ../F1/intf/2023028_2023040/corr.grd not found
[Session merge_swath (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grids ../F1/intf/2023028_2023040/corr.grd 
Number of Files to be merged is 3 
Reading in the grids...
merge_swath [ERROR]: File ../F1/intf/2023028_2023040/mask.grd not found
[Session merge_swath (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grids ../F1/intf/2023028_2023040/mask.grd 
Merging END

grdinfo [ERROR]: Error for input file: No such file (phasefilt.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files

Usage: landmask.csh region_cut[0/10600/0/27648]

    make a landmask in radar coordinates 
NOTE: The region_cut can be specified in batch.config file

SNAPHU.CSH - START
threshold_snaphu: 0.12
grdmath [ERROR]: Syntax error: corr_patch.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: corr_patch.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: mask2_patch.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: mask2_patch.grd is not a number, operator or file name
Setting search radius to be 300 ...
Reading in original grid...
nearest_grid [ERROR]: File phase_tmp.grd not found
[Session nearest_grid (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open grdfile phase_tmp.grd 
mv: cannot stat 'tmp.grd': No such file or directory
grd2xyz [ERROR]: File phase_tmp.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
grd2xyz [ERROR]: File corr_tmp.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
unwrapping phase with snaphu - higher threshold for faster unwrapping 
grdinfo [ERROR]: Error for input file: No such file (phase_patch.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files

snaphu v2.0.4
32 parameters input from file snaphu.conf.brief (297 lines total)
not enough input arguments.  type snaphu -h for help
grdinfo [ERROR]: Error for input file: No such file (phase_patch.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
grdinfo [ERROR]: Error for input file: No such file (phase_patch.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
xyz2grd [ERROR]: Error for input file: No such file (unwrap.out)
xyz2grd [ERROR]: Syntax error: Must specify -R option
xyz2grd [ERROR]: Syntax error -I option: Must specify positive increment(s)
grdinfo [ERROR]: Error for input file: No such file (phase_patch.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
grdinfo [ERROR]: Error for input file: No such file (phase_patch.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
xyz2grd [ERROR]: Error for input file: No such file (conncomp.out)
xyz2grd [ERROR]: Syntax error: Must specify -R option
xyz2grd [ERROR]: Syntax error -I option: Must specify positive increment(s)
grdmath [ERROR]: Syntax error: tmp.grd is not a number, operator or file name
mv: cannot stat 'tmp.grd': No such file or directory
grdgradient [ERROR]: Error for input file: No such file (unwrap.grd)
grdgradient [ERROR]: Syntax error: Must specify input file
grdinfo [ERROR]: Error for input file: No such file (unwrap.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
makecpt [ERROR]: Syntax error -T: Must specify valid min[/max[/inc[<unit>|+n]]] option
grdinfo [ERROR]: Error for input file: No such file (unwrap.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
grdinfo [ERROR]: Error for input file: No such file (unwrap.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
grdimage [ERROR]: Error for input file: No such file (unwrap.grd)
grdimage [ERROR]: Syntax error -I: Requires a valid grid file or a constant
grdimage [ERROR]: Syntax error: Must specify one (or three) input file(s)
grdimage [ERROR]: Syntax error -I option: Must specify intensity file, value, or modifiers
psscale [ERROR]: Could not parse nwrap.grd into Geographical, Cartesian, or Temporal coordinates!psscale [ERROR]: Syntax error -R option. Correct syntax:
	-R<xmin>/<xmax>/<ymin>/<ymax>[/<zmin>/<zmax>]
	  Append r if giving lower left and upper right coordinates
	-Rg or -Rd for global domain
	-R<grdfile> to take the domain from a grid file
psscale [ERROR]: Offending option -Runwrap.grd
psconvert [ERROR]: The file unwrap.ps has no BoundingBox in the first 20 lines or last 256 bytes. Use -A option.
Unwrapped phase map: unwrap.pdf
SNAPHU.CSH - END

GEOCODE-START
proj_ra2ll.csh
grd2xyz [ERROR]: File phasefilt.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
gmtinfo [ERROR]: No input data found!
surface [ERROR]: Syntax error: Must specify -R option
surface [ERROR]: Syntax error: Must specify -R option
grdtrack [ERROR]: Error for input file: No such file (raln.grd)
grdtrack [ERROR]: Error for input file: No such file (ralt.grd)
grdtrack [ERROR]: Syntax error: Must specify -G at least once
Sampling in geocoordinates with 50 meter pixels ...
gmtinfo [ERROR]: No input data found!
range: Subscript out of range.
gmtmath [ERROR]: Syntax error: Operation "DIV" requires 2 operands
gmtmath [ERROR]: Syntax error: Operation "MUL" requires 2 operands
gmtinfo [ERROR]: No input data found!
blockmedian [ERROR]: Unable to decode  as a floating point number
xyz2grd [ERROR]: Unable to decode  as a floating point number
grdmath [ERROR]: Syntax error: corr.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: phasefilt.grd is not a number, operator or file name
proj_ra2ll.csh
grd2xyz [ERROR]: File phasefilt_mask.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
gmtinfo [ERROR]: No input data found!
surface [ERROR]: Syntax error: Must specify -R option
surface [ERROR]: Syntax error: Must specify -R option
grdtrack [ERROR]: Error for input file: No such file (raln.grd)
grdtrack [ERROR]: Error for input file: No such file (ralt.grd)
grdtrack [ERROR]: Syntax error: Must specify -G at least once
Sampling in geocoordinates with 50 meter pixels ...
gmtinfo [ERROR]: No input data found!
range: Subscript out of range.
gmtmath [ERROR]: Syntax error: Operation "DIV" requires 2 operands
gmtmath [ERROR]: Syntax error: Operation "MUL" requires 2 operands
gmtinfo [ERROR]: No input data found!
blockmedian [ERROR]: Unable to decode  as a floating point number
xyz2grd [ERROR]: Unable to decode  as a floating point number
proj_ra2ll.csh
grd2xyz [ERROR]: File corr.grd not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
gmtinfo [ERROR]: No input data found!
surface [ERROR]: Syntax error: Must specify -R option
surface [ERROR]: Syntax error: Must specify -R option
grdtrack [ERROR]: Error for input file: No such file (raln.grd)
grdtrack [ERROR]: Error for input file: No such file (ralt.grd)
grdtrack [ERROR]: Syntax error: Must specify -G at least once
Sampling in geocoordinates with 50 meter pixels ...
gmtinfo [ERROR]: No input data found!
range: Subscript out of range.
gmtmath [ERROR]: Syntax error: Operation "DIV" requires 2 operands
gmtmath [ERROR]: Syntax error: Operation "MUL" requires 2 operands
gmtinfo [ERROR]: No input data found!
blockmedian [ERROR]: Unable to decode  as a floating point number
xyz2grd [ERROR]: Unable to decode  as a floating point number
grdinfo [ERROR]: Error for input file: No such file (corr.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
makecpt [ERROR]: Syntax error -T: Must specify valid min/max/inc[<unit>|+n] option
grdinfo [ERROR]: Error for input file: No such file (phasefilt_ll.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
gmtmath [ERROR]: Syntax error: Operation "INV" requires 1 operands
grdimage [ERROR]: Error for input file: No such file (phasefilt_ll.grd)
grdimage [ERROR]: Syntax error: Must specify one (or three) input file(s)
Make phasefilt_ll.kml and phasefilt_ll.png
psconvert [ERROR]: The file phasefilt_ll.ps has no BoundingBox in the first 20 lines or last 256 bytes. Use -A option.
grdinfo [ERROR]: Error for input file: No such file (phasefilt_mask_ll.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
gmtmath [ERROR]: Syntax error: Operation "INV" requires 1 operands
grdimage [ERROR]: Error for input file: No such file (phasefilt_mask_ll.grd)
grdimage [ERROR]: Syntax error: Must specify one (or three) input file(s)
Make phasefilt_mask_ll.kml and phasefilt_mask_ll.png
psconvert [ERROR]: The file phasefilt_mask_ll.ps has no BoundingBox in the first 20 lines or last 256 bytes. Use -A option.
grdinfo [ERROR]: Error for input file: No such file (corr_ll.grd)
grdinfo [ERROR]: Syntax error: Must specify one or more input files
gmtmath [ERROR]: Syntax error: Operation "INV" requires 1 operands
grdimage [ERROR]: Error for input file: No such file (corr_ll.grd)
grdimage [ERROR]: Syntax error: Must specify one (or three) input file(s)
Make corr_ll.kml and corr_ll.png
psconvert [ERROR]: The file corr_ll.ps has no BoundingBox in the first 20 lines or last 256 bytes. Use -A option.
GEOCODE END
rm: No match.

Yours,
Belled Deng.

**Comment by Xiaohua-Eric-Xu** [comment]
Could you send the log in your F1/2/3 directories? The processing should've failed there.

One guess is due to the recent update of orbit files by ESA. We made a fix in the master branch.  https://github.com/gmtsar/gmtsar/discussions/629

**Reply by BelledDeng** [reply]
Thanks to your suggestion. I checked my orbit files and found the problem. Compered with the old version,the new versions undate by ESA add three blank lines in the orbit files, which made the GMTSAR failed to process.Hence,I delete all of the blank lines and succeed in getting result.
Thank you very much for your guidance.
